config/externalname.go

```diff
@@ -14,6 +14,15 @@ var externalNameConfigs = map[string]config.ExternalName{
 	// Imported by using the following format: {{name}}
 	"google_active_directory_domain": config.IdentifierFromProvider,
 
+	// alloydb
+	//
+	// Imported by using the following format: projects/{{project}}/locations/{{location}}/clusters/{{cluster_id}}
+	"google_alloydb_cluster": config.TemplatedStringAsIdentifier("cluster_id", "projects/{{ .setup.configuration.project }}/location/{{ .parameters.location }}/clusters/{{ .external_name }}"),
+	// Imported by using the following format: {{cluster}}//{{instance_id}}
+	"google_alloydb_instance": config.IdentifierFromProvider,
```

Comment:
```suggestion
	"google_alloydb_instance": config.TemplatedStringAsIdentifier("instance_id", "{{ .parameters.cluster }}/{{ .external_name }}"),
```

Could you please try this configuration, if it works it could be better.